app/controllers/sac_cas/people_controller.rb

```diff
@@ -58,4 +59,40 @@ def prepare_entries(entries)
   def tabular_params(**)
     super.merge(params.slice(:recipients, :recipient_households).permit!)
   end
+
+  # BIRTHDAY VALIDATION
```

Comment:
Ab dem Punkt wo es einen Trennlinie-Kommentar braucht würd ichs glaub in eine Domain Klasse verschieben :joy: 

Dann hast du im Controller nur noch
```
def check_birthday
  return if !entry.sac_membership_active? || birthday_year == Person.find(entry.id).birthday.year

  People::SacMemberships::BirthdayValidator.new(entry).validate!
end
```

Wäre denke ich viel cleaner. Mit separaten Methoden für `#min_year` und `#max_year` usw.